apps/app/src/ui/StripePortalLink.tsx

```diff
@@ -1,39 +1,43 @@
 import { LinkExternalIcon } from "@primer/octicons-react";
-
-import config from "@/config";
+import { clsx } from "clsx";
 
 import { Anchor, anchorClassNames } from "./Link";
 
+export const ContactLink = () => {
+  return (
+    <>
+      Contact Argos support via{" "}
```

Comment:
Is "via" correct in english? I think "through" is better.

Reply:
While "via" is technically correct in English, if it bothers you, I can use "on" instead.